InventorySalesApi/Api/Data/SkuQtyRequestInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Represents 'are products salable for requested qty' request interface.
+ *
+ * @api
+ */
+interface SkuQtyRequestInterface extends ExtensibleDataInterface
```

Comment:
Let's find the better name for it, for example, `ProductSalableForRequestedQtyInfoInterface`